### PR TITLE
Update plex-media-server to 1.6.1.3722-4955e31cf

### PR DESCRIPTION
**Pull request by vin047**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/plex-media-server.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'plex-media-server' do
-  version '1.5.6.3790-4613ce077'
-  sha256 'a94b39d162029396eb7afa9131e1209cdf2bceb8296c0b9859806b31e469ef35'
+  version '1.6.1.3722-4955e31cf'
```
Comment (commitay) [inline Casks/plex-media-server.rb]: @vin047 Is `1.6.1` publicly available or plex pass only?

https://www.plex.tv/downloads/ is still at `1.5.6` and the `appcast` hasn't been updated.

Comment (vin047) [inline Casks/plex-media-server.rb]: You're right, its a plex pass download. Thought that was the version that was being tracked in Cask though? At least, i'm pretty sure it used to be...

Comment (commitay) [inline Casks/plex-media-server.rb]: > You're right, its a plex pass download

Will `1.6.1` work without plex pass?

Comment (miccal) [inline Casks/plex-media-server.rb]: We can only use the one available via the `homepage`, which as @commitay pointed out is still at `1.5.6`.

**Pull request closed without merging**